### ISSUE TITLE
Periodic popup of command-line window on Windows 10

### ISSUE THREAD

**Issue by JosephusPaye** [wakatime/sublime-wakatime#91]
Hello 👋,

I have an issue similar to https://github.com/wakatime/sublime-wakatime/issues/12, WakaTime shows an popup window on init, and also periodically whenever it sends a heartbeat.

This is really disruptive to work and makes the plugin essentially unusable.

Digging into the source and past issues, I set `python_binary` in settings to `pythonw.exe`, as per #8. However, the issue remained.

Searching online, I found this: https://stackoverflow.com/a/1016651, and made the following temporary changes to `WakaTime.py`, which fixed the issue.

Line 28 (imports):

```py
from subprocess import Popen, STDOUT, PIPE, STARTUPINFO, STARTF_USESHOWWINDOW
```

Line 320 in `find_python_in_folder()`:

```py
startupinfo = None
if os.name == 'nt':
    startupinfo = STARTUPINFO()
    startupinfo.dwFlags |= STARTF_USESHOWWINDOW
process = Popen([path, '--version'], stdout=PIPE, stderr=STDOUT, startupinfo=startupinfo)
```

And similarly at Line 554 in `send_heartbeats()`, where `Popen()` is called.

So, a combination of `pythonw`, instead of `python` for the binary and setting `STARTF_USESHOWWINDOW` on `startupinfo` when calling `Popen()` fixes the problem.

Thanks for your time.

**Comment by JosephusPaye** [comment]
Versions: 

- Sublime Text: `3.1.1`, Build `3176`
- WakaTime: `8.1.2`
- Python: `3.6.4`

**Comment by alanhamlett** [comment]
Thanks for finding this and providing a solution! It's fixed in [v8.2.0](https://github.com/wakatime/sublime-wakatime/blob/master/HISTORY.rst#820-2018-09-30) just released.

**Comment by JosephusPaye** [comment]
Thanks!
